models/grading/fact_grade_status.sql

```diff
@@ -0,0 +1,38 @@
+select
+    ls.org as org,
+    ls.course_key as course_key,
+    ls.actor_id as actor_id,
+    courses.course_name as course_name,
+    courses.course_run as course_run,
+    state as state,
```

Comment:
Would it make more sense to default to showing `failed` if there's no state for this learner yet?